code/src/sixsq/nuvla/server/util/metadata.clj

```diff
@@ -90,6 +93,28 @@
   (-> resource-ns meta :doc))
 
 
+(defn get-scrud-operations
+  "Returns the operations supported by the given resource type"
```

Comment:
scrud operations are auto-discovered by checking whether `crud/xyz` multimethods are defined or not. For this to work, the call to `gen-md/generate-metadata` needs to happen after crud defmethods, which is not always the case now.